share/models/validators.py

```diff
@@ -72,6 +74,9 @@ def __call__(self, value):
                 sorted(self.__blank_refs - self.__blank_nodes)
             ])))
 
+    def __eq__(self, other):
```

Comment:
Django uses it to check whether the validator's changed and it needs to
make a migration. (Aka docs told me to
https://docs.djangoproject.com/en/1.9/topics/migrations/#adding-a-deconstruct-method
)

On Aug 12, 2016 12:13 PM, "Chris Seto" <notifications@github.com> wrote:

> In share/models/validators.py
> <https://github.com/CenterForOpenScience/SHARE/pull/357#discussion_r74617990>
> :
>
> > @@ -72,6 +74,9 @@ def __call__(self, value):
> >                  sorted(self.__blank_refs - self.__blank_nodes)
> >              ])))
> >
> > +    def __eq__(self, other):
>
> When is this used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CenterForOpenScience/SHARE/pull/357/files/2a742809fe679d75a9f1403bbb43199a45d59ced#r74617990>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGdlftszrqDlrvGuzjsTDZ0ndfBPhURBks5qfJu0gaJpZM4JjOg6>
> .
>